lib/src/generator/templates.renderers.dart

```diff
@@ -0,0 +1,149 @@
+// ignore_for_file: camel_case_types, unused_element
```

Comment:
Also, the `checkBuild` grinder that verifies generated files are up to date before pushes should be updated to reflect this.

Reply:
Thanks! Done.